packages/record-tuple-polyfill/src/index.js

```diff
@@ -16,3 +16,4 @@
 
 export { Record } from "./record";
 export { Tuple } from "./tuple";
+export { stringify } from "./json";
```

Comment:
Wondering if we should export a `JSON` namespace here additionally... That way usage would be:

```js
import { JSON } from "polyfill";

JSON.stringify(#{ foo: 42 }); // => {"foo":42}